generic/tests/os_update.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+
+def run(test, params, env):
+    """
+    Runs yum update and yum update kernel on RHEL or apt
+    update and apt upgrade on Ubuntu based multi VM environment.
+
+    :param test: kvm test object.
+    :param params: Dictionary with test parameters.
+    :param env: Dictionary with the test environment.
+    """
+    vms = env.get_all_vms()
+    timeout = int(params.get("os_update_timeout"))
+    for vm in vms:
+        if not vm.is_alive():
+            vm.start()
+        cmd = "yum update -y && yum update kernel -y"
```

Comment:
I think that only having `"yum update -y"` is ok.

Reply:
sure, I tried to keep not to change the existing one. I can do that.